drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableViewImpl.java

```diff
@@ -153,6 +166,11 @@ private Group makeHeaderCaption() {
         return g;
     }
 
+    private double getHeaderCaptionWidth() {
```

Comment:
Here's another change calculating the "caption" width as the sum of the "Row Number" and "Description" column widths from the underlying model.